ecosystem/sep-0030.md

```diff
@@ -78,6 +83,9 @@ A client can also use the endpoints in the specification to:
 - Find out if a new signing key is available due to signing key rotation.
 - Delete accounts.
 
+Notes:
+1. <a name="note1"></a>See [Endpoints `POST /accounts/<address>/sign/<signing-address>`](#post-accountsaddresssignsigning-address) for more details.
```

Comment:
After reading this endpoint description, it is still not completely clear to me how this section provides more details to line 64 and 68. This endpoint appears to sign transactions containing signer adjustments, but its unclear what the explicit connection is.

Is the concern that an account being authorized (made a signer) for another account shouldn't be used as the source account for any operations being described above?

Reply:
The connection is the paragraph being added about what accounts a server must authorize.